docs/config.toml

```diff
@@ -75,18 +75,10 @@ weight = 1
 [outputs]
 section = ["HTML", "print"]
 
-
-[[menu.main]]
-    name = "Community"
-    weight = 40
-    url = "/docs/community"
-    pre = "<i class='fas fa-user-friends'></i>"
-
 [[menu.main]]
-    name = "GitHub"
+    name = "Download on GitHub"
```

Comment:
Why the name change here? Seems like if we were to have a download link, it would go to https://github.com/aws/eks-anywhere/releases but the actual github link is useful for more people

Reply:
I agree "GitHub" makes more sense. Verified with designer that we can keep it the way it was.